pyproject.toml

```diff
@@ -7,9 +7,19 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.9"
+scipy = ">=1.11.4"
+matplotlib = ">=3.8.2"
+numpy = ">=1.26.3"
+pandas = ">=2.1.4"
+altair = ">=5.2.0"
 
 [tool.poetry.dev-dependencies]
+
+[tool.poetry.group.dev.dependencies]
+pytest = ">=7.4.4"
+pytest-cov = ">=4.1.0"
+
```

Comment:
```suggestion
cookiecutter= ">=2.5.0"
```

Reply:
I am unsure whether we really need to list that. It is not listed in here exemplary packages...

Reply:
I think it makes sense to add becausse we use cookiecutter to generate the files?

Reply:
Hi @sivakornchong please correct me if I am wrong, but I think cookiecutter is necessary for creating the files, but after that it is not necessary for the package and thus there is no need to list it here. I have looked at the same package repositories and I have not seen it, but feel free to add it, if you want to.

Reply:
I am trying to take a look at other repository and our tutorial also. It seems that we do not need to list cookiecutter. It might because we use this package to structure our repository but if someone install our package they do not need to import this one.

Reply:
Ahh ok got it. Thanks for checking guys!

Reply:
Thank you @carinaya! Yeah, I think so too :D. Should we merge it?